ocs_ci/ocs/node.py

```
@@ -55,6 +57,149 @@
+            str: Output of the script execution.
+
+        Raises:
+            CommandFailed: In case the script execution fails.
+
+        """
+        log.info(
+            f"Running the script {script_path} with the args '{args}' "
+            f"on the worker node {self.resource_name}"
+        )
+        # Build the command — single element, no single quotes inside
+        cmd = f"set -euo pipefail; " f"bash {script_path} {args} "
```

Ruff (0.15.12)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)